.github/ISSUE_TEMPLATE/failing_test.yml

```
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Failing Test
+description: Report a failing test
+title: "[Failing Test]: "
+labels: ["bug", "awaiting triage", "test-failures", "P1"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report this test failure!
```

```suggestion
        Thanks for taking the time to report this test failure! Once you've created an issue, you can self-assign by commenting `.take-issue` and you can close by commenting `.close-issue`.
```

We recently added this to the other templates